<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INTUITA278485" name="com.geico.tests.geicoHomeTest" tests="1" failures="0" timestamp="24 Nov 2016 15:12:05 GMT" time="1.044" errors="1">
  <testcase name="verifyHome" time="1.044" classname="com.geico.tests.geicoHomeTest">
    <error type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: verifyHome([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)54481,(java.lang.String)Rojas,(java.lang.String)Cyril,(java.lang.String)10017 NYC,(java.lang.String)703,(java.lang.String)2,(java.lang.String)23,(java.lang.String)1984,(java.lang.String)2011,(java.lang.String)JAGUAR,(java.lang.String)XJL]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: verifyHome([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)54481,(java.lang.String)Rojas,(java.lang.String)Cyril,(java.lang.String)10017 NYC,(java.lang.String)703,(java.lang.String)2,(java.lang.String)23,(java.lang.String)1984,(java.lang.String)2011,(java.lang.String)JAGUAR,(java.lang.String)XJL]
	at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:52)
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1225)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1123)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- verifyHome -->
</testsuite> <!-- com.geico.tests.geicoHomeTest -->
